ios/Rainbow/AppDelegate.mm

```diff
@@ -88,9 +88,9 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                              object:nil];
   
   BOOL success = [super application:application didFinishLaunchingWithOptions:launchOptions];
-  BOOL isDetox = [[[NSProcessInfo processInfo] arguments] containsObject:@"-IS_TEST"];
+  BOOL isE2E = [[[NSProcessInfo processInfo] arguments] containsObject:@"isE2ETest"];
```

Comment:
wanted to double check if we need to keep the `-` in front of `isE2ETest` for this?

Reply:
Double checked and that works, I guess it was just detox that was adding a `-` in front.